### PR TITLE
small Makefile fixes

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fix a couple of errors in #57

### CODE REVIEW

**Comment by yaronf** [comment]
while we're at it, could you also fix it so that "make" ends with a single XML file, which is also the one that needs to be submitted?

**Comment by thomas-fossati** [comment]
> while we're at it, could you also fix it so that "make" ends with a single XML file, which is also the one that needs to be submitted?

mmm, I tend to use `make` regularly to update the TXT view --in a twin window-- of the MD file I'm editing.

Maybe we should only do the XML on a `submit` target instead?  This looks like the only time we are going to need the XML.

**Comment by yaronf** [comment]
What I meant is, when you run a full make, the result is **two** XML files. This is confusing when it comes time to submit.

I am fine with creating the XML file only for the "submit" target.

**Comment by thomas-fossati** [comment]
> What I meant is, when you run a full make, the result is **two** XML files. This is confusing when it comes time to submit.

I'd completely misunderstood, sorry!

> I am fine with creating the XML file only for the "submit" target.

I'll leave this for another day :-)

File: STAR-Delegation/Makefile [hunk 1]
```diff
@@ -14,11 +14,13 @@ ifeq ($(strip $(svgcheck)),)
 $(error svgcheck not found. To install svgcheck: 'pip install svgcheck')
 endif
 
-kdrfc_check := $(shell gem list --installed kramdown-rfc2629 -v '>= 1.3.3')
-ifeq ($(kdrfc_check), true)
-kdrfc := kdrfc -3hp
-else
-$(error kdrfc not at the right version. To install it: 'gem install kramdown-rfc2629 -v '>= 1.3.3'')
+ifeq ($(strip $(kdrfc)),)
```
Comment (thomas-fossati) [inline STAR-Delegation/Makefile]: This is a small refactoring to allow callers to supply their own `kdrfc` -- consistent with the `?=` philosophy.

File: STAR-Delegation/Makefile [hunk 2]
```diff
@@ -57,7 +60,6 @@ clean:
 	-rm -f $(draft)-[0-9][0-9].txt
 	-rm -f $(draft)-[0-9][0-9].md
 	-rm -f metadata.min.js
-	-rm -f $(art_svg)
```
Comment (yaronf) [inline STAR-Delegation/Makefile]: Are you sure? I think the SVG files *should* be removed on `make clean`.

Comment (thomas-fossati) [reply STAR-Delegation/Makefile]: The SVGs rendering of the corresponding ascii-art file is created on the fly and inlined into the XML and HTML by kdrfc.  They don't exist on disk anymore.

**Pull request merged**